pkg/api/admin_users.go

```diff
@@ -384,8 +385,15 @@ func (hs *HTTPServer) AdminLogoutUser(c *contextmodel.ReqContext) response.Respo
 		return response.Error(http.StatusBadRequest, "id is invalid", err)
 	}
 
-	if c.UserID == userID {
-		return response.Error(400, "You cannot logout yourself", nil)
+	namespace, identifier := c.SignedInUser.GetNamespacedID()
+	if namespace == identity.NamespaceUser {
```

Comment:
Shouldn't we return a bad request if the namespace is not user?

Reply:
In this case it's not an issue if the active user is not a User. Only if the target user is also the active user :sweat_smile: . This is kinda hard to explain verbosely